blobstore/src/main/java/org/jclouds/blobstore/options/ListContainerOptions.java

```diff
@@ -231,6 +244,15 @@ public ListContainerOptions delimiter(String delimiterString) {
       return this;
    }
 
+   /**
+    * return a listing of all objects inside the store, recursively.
+    */
+   public ListContainerOptions versions() {
+      // checkArgument(path == null, "path and recursive combination currently not supported");
```

Comment:
commented out code

Reply:
Will be removed as part of bucket configuration PR